connector_magento/components/backend_adapter.py

```diff
@@ -86,31 +89,27 @@ def __init__(
             client_secret=consumer_secret,
             resource_owner_key=resource_owner_key,
             resource_owner_secret=resource_owner_secret,
-            signature_method=SIGNATURE_HMAC_SHA256,
```

Comment:
do we need to be mindful of a changing default/provide a mechanism to change this incase the Magento dev team changes something?

I'm unclear here so I'll yield to your opinion :)